src/components/footer/Footer.tsx

```diff
@@ -0,0 +1,38 @@
+import Box from '@mui/material/Box';
+import Container, { ContainerProps } from '@mui/material/Container';
+import Stack from '@mui/material/Stack';
+import Typography from '@mui/material/Typography';
+
+import { footerStyles } from './footerStyles';
+import { useTheme } from '@mui/material';
+import { FOOTER_BG_LIGHT, PRIMARY_LIGHT_THEME } from '../../theme/themeTokens';
+import { MuiThemeMode } from '../../types/theme.types';
+
+export const Footer = ({ maxWidth }: { maxWidth: ContainerProps['maxWidth'] }) => {
+  const {
+    palette: { mode },
+  } = useTheme();
+
+  return (
+    <Box
+      sx={footerStyles.footer}
+      bgcolor={mode === MuiThemeMode.light ? FOOTER_BG_LIGHT : PRIMARY_LIGHT_THEME}
```

Comment:
look strange. FOOTER_BG_LIGHT and PRIMARY_LIGHT_THEME footer color should be already provided in the themeProvider (fields in lighttherme and darktheme)